Qase_Test/Src/Utils/Waiters.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using Qase_Test.Core;
+
+namespace Qase_Test.Utils
+{
+    public class Waiters
```

Comment:
Would be better to define it as a static class. So in this case it will be not necessary to pass BrowserService in each page constructor